app/models/protocol.rb

```diff
@@ -254,11 +254,7 @@ def self.viewable_by_user(user, teams, options = {})
   end
 
   def self.viewable_by_user_my_module_protocols(user, teams)
```

Comment:
We don't go below tasks with permissions, so it is not possible to assume that such protocols have user assignments. 
And this method is for linked to tasks protocols